### DISCUSSION TITLE
[Feature] show details when resolving duplicates

### DISCUSSION THREAD

**Discussion by krzemyczek** [immich-app/immich#14844] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have many photos with incorrect date and the same photos with the correct one (thanks Google). Right now I have to hover to photo and find the photo date in this:
![image](https://github.com/user-attachments/assets/47ece144-c0bf-4f85-a1ce-8357c7d99bb2)
which is... time consuming
For me, more sens have displaying the photo date other than album (or "Not in any album")

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by scottcopus** [comment]
Good ideas. I'd like to expand on this deduplicate enhancement request to add "**originalPath**" to the thumbnail preview tiles too.  Here's my use case:  I have about 20K assets in external libraries and Immich has identified about 10% of them to be duplicates.  FYI, I never had a good import strategy until a few years ago until I found PhotoSync.  My duplicates are typically in different folders though.  Some of these duplicates have gone thru various tools over time, such as red-eye reduction.  These folder names can help me identify which ones I'd like to keep versus trash.  And more importantly... to keep my external libraries as organized as possible.  This means I would typically prioritize which images from certain folders to trash versus keep.  So the more info that it shown (by default--without looking at the hover info) in the dedupe process would be immensely helpful.

Maybe it would be a good idea to have "options" for the dedupe 'app'  where one could choose which of the details would appear by default?

Keep up the great work!

cheers.

**Reply by krzemyczek** [reply]
Maybe even not in Immich itself, but something similar to immich-go which has a ton of features, but not enough to help with many use cases. However, I'm afraid that this is absolutely out of scope of Immich team because of complication of this :c

**Reply by jmigual** [reply]
Have you seen this: https://github.com/vale46n1/immich_duplicate_finder ?
